docs/copied-from-beats/dashboards.asciidoc

```diff
@@ -32,42 +32,54 @@ the _Beats Developer Guide_.
 ifndef::only-elasticsearch[]
 If you've configured the Logstash output, see
 <<load-dashboards-logstash>>.
-endif::[]
-
-ifdef::allplatforms[]
+endif::only-elasticsearch[]
 
 ifeval::["{requires-sudo}"=="yes"]
 
 include::../../libbeat/docs/shared-note-sudo.asciidoc[]
 
 endif::[]
 
+ifdef::deb_os,rpm_os[]
 *deb and rpm:*
 
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
 {beatname_lc} setup --dashboards
 ----------------------------------------------------------------------
+endif::deb_os,rpm_os[]
 
-
+ifdef::mac_os[]
 *mac:*
 
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
 ./{beatname_lc} setup --dashboards
 ----------------------------------------------------------------------
+endif::mac_os[]
 
+ifdef::linux_os[]
```

Comment:
Do we want to include `linux_os`?

Reply:
is it excluded in 1527? then it might be a mistake, we want `linux_os`

Reply:
updated with 5aac4bd